### DISCUSSION TITLE
Welcome to content-collections Discussions!

### DISCUSSION THREAD

**Discussion by sdorra** [sdorra/content-collections#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by malikpiara** [comment]
Hi! I just recently came across this project when I was looking for a replacement for ContentLayer.
Using it now with MDX on my personal blog ([example](https://moonwith.com/posts/26)).

What a great work and service @sdorra! What are your plans for the future of Content Collections?

**Reply by sdorra** [reply]
Thank you for your kind words and sponsorship. My next major goal is to work on documentation. I keep track of my tasks in the [issue](https://github.com/sdorra/content-collections/issues). @malikpiara, is there anything you miss?

**Reply by malikpiara** [reply]
ContentLayer's NextJS ready-to-use template was a main reason why I ended up using it. And I believe creating similar would lead to a great increase in user adoption and support from the community.

I'm not a seasoned developer but I absolutely love product work, from discovery to defining requirements to making decisions about what should be prioritised, and haven't yet had a chance to do product management for something more technical. If you feel you'd like support there, I'd be overjoyed! Happy to invite you for coffee and co-working in Berlin either way!

**Reply by sdorra** [reply]
@malikpiara, I'm unsure about providing templates since I support various frameworks for Content Collections and don't want to favor one. Creating templates for all is time-consuming to maintain. Instead, a CLI command could be developed to install and set up Content Collections in a project (#32) using (npx or pnpx). This approach would involve two commands rather than a single one. What are your thoughts?

**Reply by sdorra** [reply]
> I'm not a seasoned developer but I absolutely love product work, from discovery to defining requirements to making decisions about what should be prioritised, and haven't yet had a chance to do product management for something more technical. If you feel you'd like support there, I'd be overjoyed! Happy to invite you for coffee and co-working in Berlin either way!

I appreciate any help. Thanks for offering coffee, but Berlin is too far for me.

**Reply by sdorra** [reply]
@malikpiara is [next-contentlayer-example](https://github.com/contentlayerdev/next-contentlayer-example) the template you used?

**Reply by malikpiara** [reply]
@sdorra No. I used this one: https://github.com/shadcn/next-contentlayer
